### DISCUSSION TITLE
playbook not indexing correctly

### DISCUSSION THREAD

**Discussion by ktroberson** [Security-Onion-Solutions/securityonion#8356] [Unsupported Versions] [answered]
Good morning all. Quick question. We are working on an older version of security onion during an exercise and we have playbook running and triggering alerts. But the fields are coming back not searchable. I have tried going into index patterns and refreshing, but no luck. I have attached a screen shot. Any help would be grateful. We are using Sec onion ver 2.3.21
![Capture](https://user-images.githubusercontent.com/29292838/180251713-3ade903b-12fe-452b-97f7-3c117b3fe8ea.PNG)

**Comment by defensivedepth** [comment]
hello there!

Can you confirm which version of SO you are running? 

Also, when you say `not searchable`, can you describe what you mean by this a bit more?

**Comment by ktroberson** [comment, marked as answer]
We are using Sec onion ver 2.3.21. As you can see in my screen shot with the yellow triangles those fields are not abled to be used to search against.

**Reply by defensivedepth** [reply]
Thanks for confirming!

As you mentioned, this is a very old version of SO v2 (Dec 2020). This issue was fixed quite a while ago... iirc it had to do with adding `event_data` to the template that the Playbook Alerts index uses.

To be clear, all the data data that is under `event_data` are the original fields / data from the log that generated the alert. You should still be able to find that original log and then search directly off of that. You might also try using Hunt.
